Tutorials/Creating-Tables-for-Umbraco-with-PetaPoco/index-v7.md

```diff
@@ -0,0 +1,248 @@
+---
+versionFrom: 7.0.0
+---
+# Tutorial - Creating Tables in Umbraco with PetaPoco
+
+## Overview
+
+This guide will take you steps to create tables in Umbraco using Peta Pocos and the ability to access that data in that table
+
+## What is Peta Poco?
+
+PetaPoco is a tiny & fast micro-ORM for . NET. It is tiny with abosoutely no dependencies
+
+## Requirements Needed to Use PetaPoco
+
+- Umbraco 7+ with a SQL Database
+- Umbraco 8+ does not support Peta Poco
```

Comment:
Instead of mentioning this here, it would be great to add the a YAML tag instead at the top.
So, under the `versionFrom` tag, add the `versionRemoved` to make it clear that this tutorial cannot be used for Umbraco 8.

```suggestion
```